node/task_4/main.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('file-system');
+const currentDate = require('./date');
+const dir_name = 'date_dir';
+const file_name = 'current_date';
```

Comment:
we're using camelCase syntax, please use it further. It is a convention for our team